src/server/auth/server/api_server.go

```diff
@@ -872,6 +872,22 @@ func (a *apiServer) GetAuthTokenInTransaction(txnCtx *txnenv.TransactionContext,
 	}, nil
 }
 
+// GetPipelineAuthTokenInTransaction is an internal API used to create a pipeline token for a given pipeline.
+// Not an RPC.
```

Comment:
Maybe add an extra note about how it's insecure? Something like `does not authenticate the caller, and should only be used in code paths that have already authenticated the caller (currently just CreatePipeline)` or something?